<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListarUsuario">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar Usuario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRealizarEmprestimo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Realizar Emprestimo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLivro">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar Livro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCadastrarLivro">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastrar Livro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCadastrarUsuario">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastrar Usuario</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
